core/components/blocks/Cms/Block.js

```diff
@@ -0,0 +1,41 @@
+export default {
```

Comment:
core components folder is now depreciated (pull latest dev), please use it in appropiate module